src/common/config/env.validation.ts

```diff
@@ -96,3 +96,21 @@ export function validate(config: Record<string, unknown>) {
 
   return validatedConfig;
 }
+
+export const ENV_KEYS = [
```

Comment:
use `Reflect.ownKeys(new EnvironmentVariables())`

Reply:
it is not equal, it doesnt return keys which value are undefined.
And `new EnvironmentVariables() ` has many undefined, basically all that doesnt have default value.

I was playing with that and the at end appeared to much code changes and hacks around (adding `@Expose()`, reworking default values and using `plainToInstance`).

And I just clean up all these changes and made simple array.

Reply:
I tested it in browser console. I created same object with undefined keys and it worked.